lib/matplotlib/tests/test_gridspec.py

```diff
@@ -35,3 +36,16 @@ def test_repr():
                            width_ratios=(1, 3))
     assert repr(ss) == \
         "GridSpec(2, 2, height_ratios=(3, 1), width_ratios=(1, 3))"
+
+
+def test_subplotspec_args():
+    fig, axs = plt.subplots(1, 2)
+    # should work:
+    gs = gridspec.GridSpecFromSubplotSpec(2, 1,
+                                          subplot_spec=axs[0].get_subplotspec())
+    assert gs.get_topmost_subplotspec() == axs[0].get_subplotspec()
+    # this is a mistake, and not what the type hints give, but we allow:
```

Comment:
Is this comment from an earlier revision?  It says we allow, but then the code asserts an exception.

Reply:
Fixed